pkgs/development/libraries/kde-frameworks/kapidox.nix

```diff
@@ -1,9 +1,25 @@
-{ mkDerivation, lib, extra-cmake-modules, python3 }:
+{ mkDerivation, python3, qtbase }:
 
 mkDerivation {
```

Comment:
I would be happier if we used buildPythonApplication.

Reply:
Me too, but not sure how that'd pan out with kde's custom `mkDerivation` (which merges these attributes with `srcs.nix` and versions). Would you have a suggestion on how to integrate `buildPythonApplication` with that?

Reply:
Add it next to https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/kde-frameworks/default.nix#L92-L94 ?

Reply:
Is the trouble worth it? Not like we build several python libraries/applications for all supported python environments/versions, the build is just a couple of `setup.py` commands, only enough to satisfy this marginally required dependency. Also, I tried it a little, and my Nix knowledge may not be enough, as it may involve passing the whole attrset of pythonPackages as parameters of the derivation, as is done with the `libsForQt5` set, but I'm not sure about the safe way to do this with both in the same call.

Reply:
Alternately, we could just drop this package. It's never been used in Nixpkgs. I'm not even sure it has ever been buildable before.